docs/content/guides/dagster/automated_pipelines.mdx

```diff
@@ -0,0 +1,157 @@
+---
+title: Automating Your Data Pipelines | Dagster
+description: This guide illustrates how to automate your data pipelines 
+---
+
+# Automating your data pipelines
+
+In this guide, we’ll walk you through how to run your data pipelines without manual intervention, i.e. automate, and identify the Dagster tools to make that happen.
+
+This guide assumes you have some familiarity with several Dagster concepts, including [software-defined assets,](https://docs.dagster.io/concepts/assets/software-defined-assets) [ops](https://docs.dagster.io/concepts/ops-jobs-graphs/ops) and [graphs](https://docs.dagster.io/concepts/ops-jobs-graphs/graphs), and [jobs](https://docs.dagster.io/concepts/ops-jobs-graphs/jobs), and [when to use each](https://docs.dagster.io/guides/dagster/how-assets-relate-to-ops-and-graphs).
+
```

Comment:
The structure of this guide is something like "if I have X need, I should do Y", and I think it's worth explicitly spelling that out.  Something like: "Dagster offers a few different ways of automating data pipelines, and choosing the right one depends on your specific needs. In this guide we'll walk through a few different cases, and which Dagster tools you should use in each case."